integrations/jenkins/Jenkinsfile

```diff
@@ -387,6 +393,13 @@ pipeline {
             }
 
             steps {
+                script {
+                    params.ENVIRONMENT_VARIABLES.tokenize(',').each {
+                        def (key, value) = it.tokenize('=')
```

Comment:
Should this only split after the first `=` to allow the use of `=` within the string value of an environment variable?